core/include/aes_gcm_dev.h

```diff
@@ -1,16 +1,16 @@
 #pragma once
 
-#include <squareup/subzero/internal.pb.h>
+/* This header is meant for the dev target, and should only be included in the
+ * dev implementation */
+
+// M_KeyID typedef for the dev target
```

Comment:
Go ahead and add a pre-emptive comment here explaining that it is 1 based since the dev target has pre-initialized values and 0 represents uninitialized. As a new viewer, it wasn't clear why this wasn't 0 like in the ncipher version.

Reply:
Will do. You are not the first one to have questions about this, and I agree we should be as explicit as possible.